### ISSUE TITLE
FLET with an empty body fails to compile.

### ISSUE THREAD

**Issue by stassats** [drmeister/clasp#207]
```
(defun foo ()
  (flet ((bar ()))))
```

```
The assertion (AND
               (NULL
                (CLEAVIR-AST-TO-HIR::RESULTS
                 CLEAVIR-AST-TO-HIR:CONTEXT))
               (=
                (LENGTH
                 (CLEAVIR-AST-TO-HIR::SUCCESSORS
                  CLEAVIR-AST-TO-HIR:CONTEXT))
                1)) failed
```

**Comment by Shinmera** [comment]
Cannot confirm on b8c6690535bdc61bfdaeb50dbe23cd89633200a5
```
* (defun foo ()
  (flet ((bar ()))))

FOO
* (compile NIL '(lambda () (flet ((bar ())))))
; in: LAMBDA ()
;     (FLET ((BAR NIL)))
; 
; note: deleting unused function
;   (FLET BAR)
; 
; compilation unit finished
;   printed 1 note

#<FUNCTION (LAMBDA ()) {1004F1EEDB}>
NIL
NIL
```

**Comment by Bike** [comment]
This should be fixed by https://github.com/robert-strandh/SICL/commit/b9e2404ffa686c3f6e73121770043c860cfbd019

**Comment by Shinmera** [comment]
Can confirm fixed on 1b63a29f869513c9fec111b5963d98aa0503155b
